tests/gcp/installer/util/test_resource/forseti_conf_server.yaml.in

```diff
@@ -0,0 +1,244 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is an input file that is used by the Forseti prerequisites setup
+# wizard to generate an actual forseti_conf.yaml. It's better to work off
+# the forseti_conf.yaml.sample than this one if you are editing the config
+# by hand.
+
+global:
```

Comment:
Hmmm, is this test copy of the yaml.in file necessary?  Can we not re-use the actual yaml.in file for testing?

Reply:
I think it's better to separate them. We are testing the functionality of one of the functions and not the yaml file, otherwise if someone updated the .yaml.in file, he will need to also update the expected output file even thought the functionality of the function didn't change.